test/livebook_web/live/hub_live_test.exs

```diff
@@ -25,38 +24,40 @@ defmodule LivebookWeb.HubLiveTest do
 
       {:ok, view, _html} = live(conn, "/hub")
 
-      # renders the second step
       assert view
              |> element("#fly")
              |> render_click() =~ "2. Configure your Hub"
 
-      # triggers the access_access_token field change
-      # and shows the fly's third step
       assert view
              |> element(~s/input[name="fly[access_token]"]/)
              |> render_change(%{"fly" => %{"access_token" => "dummy access token"}}) =~
                ~s(<option value="123456789">Foo Bar - 123456789</option>)
 
-      # triggers the application_id field change
-      # and assigns `selected_app` to socket
+      attrs = %{
+        "access_token" => "dummy access token",
+        "application_id" => "123456789",
+        "hub_name" => "My Foo Hub",
+        "hub_color" => "#FF00FF"
+      }
+
       view
-      |> element(~s/select[name="fly[application_id]"]/)
-      |> render_change(%{"fly" => %{"application_id" => "123456789"}})
+      |> element("#fly-form")
+      |> render_change(%{"fly" => attrs})
+
+      refute render(view) =~ "can't be blank"
+      refute render(view) =~ "is invalid"
+      refute render(view) =~ "already exists"
+      refute render(view) =~ "does not exists"
+      refute render(view) =~ "is not a valid color"
```

Comment:
Asserting that all possible errors are not present doesn't scale. Perhaps we can use a selector to assert that the form doesn't have any errors?

Reply:
`.invalid-feedback` I guess this works pretty well